mirdata/datasets/dagstuhl_choirset.py

```diff
@@ -0,0 +1,411 @@
+"""Dagstuhl ChoirSet Dataset Loader
+
+.. admonition:: Dataset Info
+    :class: dropdown
+
+    Dagstuhl ChoirSet (DCS) is a multitrack dataset of a cappella choral music.
+    The dataset includes recordings of an amateur vocal ensemble performing two
+    choir pieces in full choir and quartet settings (total duration 55min 30sec).
+    The audio data was recorded during an MIR seminar at Schloss Dagstuhl using
+    different close-up microphones (dynamic, headset and larynx microphones) to
+    capture the individual singers’ voices.
+
+    For more details, we refer to:
+    Sebastian Rosenzweig (1), Helena Cuesta (2), Christof Weiß (1),
+    Frank Scherbaum (3), Emilia Gómez (2,4), and Meinard Müller (1):
+    Dagstuhl ChoirSet: A Multitrack Dataset for MIR Research on Choral Singing.
+    Transactions of the International Society for Music Information Retrieval,
+    3(1), pp. 98–110, 2020.
+    DOI: https://doi.org/10.5334/tismir.48
+
+    (1) International Audio Laboratories Erlangen, DE
+    (2) Music Technology Group, Universitat Pompeu Fabra, Barcelona, ES
+    (3) University of Potsdam, DE
+    (4) Joint Research Centre, European Commission, Seville, ES
```

Comment:
We didn't include more info about the dataset pieces because we have different singer configurations/mics/takes...etc., for each of them + vocal exercises. We think writing this in the description would be too lengthy but we will add a couple of sentences with some more information.

Regarding the multi-track vs multi-mic dataset: even though there's some bleeding between mics, the dataset is useful for source separation (we're actually working on SS using this dataset, it's doable). Besides, this dataset is particularly interesting for more music-focused analyses e.g., singing intonation analysis, that wouldn't be possible with a non-multitrack dataset.

Reply:
Maybe an intermediate solution - I do think this dataset would be a bit clearer to an unfamiliar user if there's a short description of the kinds of microphones that are present. What about describing each of the types of recordings for the Tracks/MultiTracks and then mentioning that not every one has every type of recording?